.github/workflows/lint-php.yml

```diff
@@ -0,0 +1,52 @@
+name: Lint PHP
+
+on:
+  workflow_call:
+    inputs:
+      PHP_MATRIX:
+        description: Matrix of PHP versions as a JSON formatted object.
+        default: '["8.0"]'
+        required: false
+        type: string
+      COMPOSER_ARGS:
+        description: Set of arguments passed to Composer.
+        default: '--prefer-dist'
+        required: false
+        type: string
```

Comment:
Composer is installed in case you want to run that script on a `*-website`-repository which needs to pull Plugins, Themes, Libaries, ...

We will introduce a  `SKIP_COMPOSER_INSTALL` input to allow skipping that step 👍🏻 See also https://github.com/inpsyde/reusable-workflows/pull/19#discussion_r995387999